rascal2/ui/view.py

```diff
@@ -37,13 +39,69 @@ def __init__(self):
         self.setMinimumSize(1024, 900)
         self.setAttribute(QtCore.Qt.WidgetAttribute.WA_DeleteOnClose)
 
+        self.startup_dlg, self.project_dlg = StartUpWidget(self), ProjectDialog(self)
+
+        self.setCentralWidget(self.startup_dlg)
+
+    def showProjectDialog(self):
+        """Shows the dialog to create a new rascal project"""
+        self.new_project_action_called = True
+        self.project_dlg = ProjectDialog(self)
+        self.project_dlg.show()
+
+    def toggleView(self):
+        """Toggles between startup widget and project dialog"""
+        self.startup_dlg.hide() if self.startup_dlg.isVisible() else self.startup_dlg.show()
+        self.project_dlg.hide() if self.project_dlg.isVisible() else self.project_dlg.show()
+
+    def createNewProject(self):
+        """Creates a new rascal project"""
+        self.setWindowTitle(MAIN_WINDOW_TITLE + " - " + self.project_dlg.project_name.text())
+        if not self.new_project_action_called:
+            self.startup_dlg.close()
+        self.project_dlg.close()
+        self.toolbar.setEnabled(True)
+        self.setupMDI()
+
```

Comment:
this function should actually create the project:
`self.presenter.model.createProject(name, save_path)`
where name and save path are grabbed from the dialog